docs/developer-guide/ksqldb-reference/expressions.md

```diff
@@ -0,0 +1,44 @@
+---
```

Comment:
This is great! Can you add a link in mkdocs.yml?

Reply:
I deleted the expressions.md. I initially was going to use it, but I ended up putting the syntax in the operators.md. I just forgot to delete the file.